src/file/file.service.spec.ts

```diff
@@ -154,6 +154,50 @@ describe('File Logic', () => {
     });
   });
 
+  describe('#createUpdate', () => {
+    it('should return a new upload', async () => {
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
+      await FileService.create(bucket, 'file.txt', USER.id, 'text', key, key2, size);
+      const newUpdate: IUpload = await UploadService.createUpdate(key, bucket, 'file.txt',  USER.id, key3, size).should.eventually.exist;
+      expect(newUpdate).to.exist;
+      expect(newUpdate.bucket).to.be.equal(bucket);
+      expect(newUpdate.name).to.be.equal('file.txt');
+      expect(newUpdate.key).to.be.equal(key);
+    });
+
+    it('should return file not exist', async () => {
+      await UploadService.createUpdate(testUpload.key, testUpload.bucket, testUpload.name, USER.id, null, 256)
+      .should.eventually.be.rejectedWith(FileNotFoundError);
+    });
+
+    it('should throw exceeded quota error', async () => {
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
+      await FileService.create(bucket, 'file.txt', USER.id, 'text', key, key2, size);
+      await UploadService.createUpdate(key, bucket, 'file.txt',  USER.id, key3, 101 * 1024 * 1024 * 1024)
```

Comment:
make the `101 * 1024 * 1024 * 1024` a constant at the beginning of the file, and call it `ExceedQuotaSize`